packages/server/src/index.ts

```diff
@@ -46,21 +47,20 @@ export interface ServerAdapterObject<
   /**
    * This function takes Node's request object and returns a WHATWG Fetch spec compliant `Response` object.
    **/
-  handleNodeRequest(nodeRequest: NodeRequest, ctx: TServerContext): Promise<Response> | Response;
+  handleNodeRequest(nodeRequest: NodeRequest, ...ctx: Partial<TServerContext>[]): Promise<Response> | Response;
   /**
    * A request listener function that can be used with any Node server variation.
    */
   requestListener: RequestListener;
-  /**
-   * Proxy to requestListener to mimic Node middlewares
-   */
-  handle: ServerAdapterObject<TServerContext, TBaseObject>['requestListener'] &
-    ServerAdapterObject<TServerContext, TBaseObject>['fetch'];
```

Comment:
I didn't want to use `fetch` signature directly, because we don't support `URL` or `string` in our generic handler.

Reply:
Makes sense!